packages/core/src/SlideBox/Slide.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useRef, useEffect } from 'react'
+import PropTypes from 'prop-types'
+import { Box } from '..'
+import { RenderInView } from './RenderInView'
+import { SlideWrapper } from './styles'
+
+interface ISlideProps {
+  onSlideChange: PropTypes.func
+  slideSpacing: PropTypes.number
+  stretchHeight: PropTypes.bool
+  index: PropTypes.number
+  content: PropTypes.node
+  isCurrentSlide: PropTypes.bool
+  width: PropTypes.any
+  numSlides: PropTypes.number
+}
+
+const Slide: React.FC<ISlideProps> = ({
+  onSlideChange,
+  slideSpacing,
+  stretchHeight,
+  index,
+  content,
+  isCurrentSlide,
+  width,
+  numSlides,
+}) => {
+  const ref = useRef()
+
+  useEffect(() => {
+    if (isCurrentSlide === true) {
+      /* istanbul ignore next */
+      ref?.current?.scrollIntoView({ block: 'end' })
+    }
+  }, [isCurrentSlide])
```

Comment:
shouldn't `ref` be a dependency to the effect?

Reply:
Updated